packages/livechat/src/widget.js

```diff
@@ -138,6 +138,7 @@ const createWidget = (url) => {
 		smallScreen = matches;
 		updateWidgetStyle(widget.dataset.state === 'opened');
 		callHook('setExpanded', smallScreen);
+		callHook('setParentUrl', window.location.url);
```

Comment:
What's the difference between `url` and `href`? On here I have access to `href` but not `url` 🤔